pkg/storage/business_case.go

```diff
@@ -81,6 +81,33 @@ func (s *Store) FetchBusinessCaseIDByIntakeID(ctx context.Context, intakeID uuid
 	return businessCaseID, nil
 }
 
+// FetchOpenBusinessCaseByIntakeID queries the DB for an open business case matching the given intake ID
```

Comment:
Do we need a way to fetch only open business cases? Maybe for the system banners? I don't think we should default our queries to only fetch the open business cases, but likely add in a filter param to the API to explicitly state whether we want just open requests, just closed requests, or all requests.